e2e/cypress/integration/account_settings/profile/profile_picture_spec.js

```diff
@@ -69,4 +69,19 @@ describe('Profile > Profile Settings > Profile Picture', () => {
             should('have.attr', 'src').
             and('not.include', customImageMatch);
     });
+
+    it('MM-T2077 Profile picture: non image file shows error', () => {
+        // # Go to Profile
+        cy.uiOpenProfileModal();
+
+        // # Click "Edit" to the right of "Profile Picture"
+        cy.get('#pictureEdit').should('be.visible').click();
+
+        // # Upload and save profile picture
+        cy.findByTestId('uploadPicture').attachFile('txt-changed-as-png.png');
+        cy.uiSave().wait(TIMEOUTS.HALF_SEC);
+
+        // # Verify error message
+        cy.get('.has-error').should('be.visible');
```

Comment:
I didnt check the error message, because in my case, error message was "resolution is too high" which is strange to me. Please guide to updated this part of the test to better cover the cases